google/cloud/pubsub/topic_admin_connection.h

```diff
@@ -158,6 +159,45 @@ class TopicAdminConnection {
   virtual ListTopicSnapshotsRange ListTopicSnapshots(ListTopicSnapshotsParams);
 };
 
+/**
+ * Creates a new `TopicAdminConnection` object to work with `TopicAdminClient`.
+ *
+ * @warning This method exists solely for backwards compatibility. It prevents
+ *     existing code, which calls `MakeTopicAdminConnection({})` from breaking,
+ *     due to ambiguity. Please use the `MakeTopicAdminConnection` method which
+ *     accepts `google::cloud::Options` instead.
```

Comment:
I should change this to say "Please use `MakeTopicAdminConnection()` instead."

(I will wait to make the edit, because I suspect there are other things that will need to change)